packages/core/src/machines/index.ts

```diff
@@ -137,7 +142,20 @@ export const createAuthMachine = ({
                   actions: ['saveSession', 'reportTokenChanged'],
                   target: 'signedIn'
                 },
-                onError: { actions: ['saveAuthenticationError'], target: 'signedOut' }
+                onError: [
```

Comment:
Could you please write tests for this?

Reply:
Yes, as soon as we will have validated the design